README.rst

```diff
@@ -497,25 +454,24 @@ If you choose to work with the main branch, we recommend using NVIDIA's PyTorch
     -p 8888:8888 -p 6006:6006 --ulimit memlock=-1 --ulimit \
     stack=67108864 --device=/dev/snd nvcr.io/nvidia/pytorch:23.10-py3
 
-Examples
---------
+Future Work
+-----------
 
-Many examples can be found under the `"Examples" <https://github.com/NVIDIA/NeMo/tree/stable/examples>`_ folder.
+The NeMo Framework Launcher does not currently support ASR and TTS training, but it will soon.
 
+Get Help
```

Comment:
Revert - it sounds very wrong to say "Get Help" - just keep it as Contributing & Discussion

Reply:
Yes, the original text was "Getting Help." How about Access the Discussions Board and Contribute to NeMo? FYI, we try and avoid the use of gerunds (ing) in headings and replace with imperative verbs for tasks.

Reply:
OK